src/cache/cache-instances.js

```diff
@@ -1,25 +0,0 @@
-const cacheDnsList = require('./cache-list');
-const config = require('../config');
-
-const getInstances = () => {
-  const env = config.clusterEnv;
-
-  if (env !== 'development') {
-    return cacheDnsList[config.awsRegion];
-  }
-
-  return {
-    master: {
-      host: 'localhost',
-      port: '6379',
-    },
-    slaves: [
-      {
-        host: 'localhost',
-        port: '6379',
-      },
-    ],
-  };
-};
-
-module.exports = getInstances;
```

Comment:
interesting, it seems like this file was never used?

Reply:
It was used, but the file it was used was deleted in this release too. The entire dependency chain was very deep to build configuration object that was not really necessary, so everything got abstracted into `generate-config.js`.

Reply:
For reference, it goes: `cache/index.js` > `cache/sanitize.js` (via `sanitiseGlobalConfiguration`), which calls `getInstances`.

Reply:
I just looked at all of them and at your implementation and I understand now. Your changes are so much better! The previous code was so hardcoded and poorly written that I wonder how it managed to make it to master in that state